### PR TITLE
Custom Rock Paper Scissors exercises: Replace Utils.feedback with TestedCell

### PR DESCRIPTION
**Pull request by Mraiih**
### Todos
- [x] replace all feedbacks with TestedCell
- [x] add solution for each TestedCell
- [x] remove all assertions in `feedback.ex`
- [x] remove all solution in  `solutions.ex`

### Screens
03_22_31_13_471_vivaldi

### CODE REVIEW

File: exercises/custom_rock_paper_scissors.livemd [hunk 1]
```diff
@@ -25,15 +40,69 @@ For example,
 game = CustomGame.new(:grass, :fire, :water)
 
 CustomGame.play(game, :fire, :water)
-":water beats :fire"
+"water beats fire"
```
Comment (BrooklinJazz) [inline exercises/custom_rock_paper_scissors.livemd:43]: I think the atom should be included (based on the tests).

I'm happy to alter it, however we should reflect the tests to match.

**Review by BrooklinJazz — CHANGES_REQUESTED** [review]
Awesome as always! There was one thing I notice with the instruction and the test not matching.

File: exercises/custom_rock_paper_scissors.livemd (hunk 1 above)
Comment (Mraiih) [reply exercises/custom_rock_paper_scissors.livemd:43]: Well, when we interpolate an atom in a string it removes the colon

Comment (BrooklinJazz) [reply exercises/custom_rock_paper_scissors.livemd:43]: TIL. Approving!

**Pull request merged**